libraries/AP_Generator/AP_Generator_IE_650_800.cpp

```diff
@@ -40,7 +40,7 @@ void AP_Generator_IE_650_800::assign_measurements(const uint32_t now)
     _err_code = _parsed.err_code;
 
     // Update variables to be returned to front end
-    _fuel_remain_pct = _parsed.tank_pct * 0.01;
```

Comment:
Discussed on the dev call and the decision was to remove the suffix or use "_proportion".  I'm happy with either these ideas.